app/src/main/java/com/cmput301w23t40/capturetheqr/MapActivity.java

```diff
@@ -19,9 +38,47 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_map);
 
+        /* Copied from
```

Comment:
Would you be able to use the following comment scheme for citing please? Example below

    /* Adapted/Copied code from the following resource for the swipe-to-delete functionality:
            author: https://auth.geeksforgeeks.org/user/chaitanyamunje
            url: https://www.geeksforgeeks.org/swipe-to-delete-and-undo-in-android-recyclerview/
            last updated: Oct 5, 2021
    */

Reply:
Done